### PR TITLE
Update types-introduction.md

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

File: javascript/core/types/types-introduction.md [hunk 1]
```diff
@@ -28,57 +28,68 @@ aspects:
...
+- `symbol`
+
+The main non-primitive type is `Object`.
 
 There are also a few special types - `Array`, `Date`, `RegExp` - that are types of `Object` but included out of the box.
 
-Unlike some other languages you do not need to tell JavaScript what type a variable is when declaring it as the interpreter will work it out for you:
+### typeof
+
+You can check the type of value by using the `typeof` operator e.g.
```
Comment (nem035) [inline javascript/core/types/types-introduction.md]:
```suggestion
You can check the type of a value by using the `typeof` operator e.g.
```

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/core/types/types-introduction.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/types/types-introduction.md?ref=814ba8c9febe0b074579b944769ee7aefa59bac3)

File: javascript/core/types/types-introduction.md [hunk 2]
```diff
@@ -28,57 +28,68 @@ aspects:
...
 
 ```javascript
-var name = "john smith";
+var company="enki";
```
Comment (nem035) [inline javascript/core/types/types-introduction.md]:
```suggestion
let company = "enki";
```

File: javascript/core/types/types-introduction.md [hunk 3]
```diff
@@ -28,57 +28,68 @@ aspects:
...
+typeof company; //string
```
Comment (nem035) [inline javascript/core/types/types-introduction.md]:
```suggestion
typeof company; // "string"
```

**Pull request merged**